.pkgmeta

```diff
@@ -0,0 +1,19 @@
+manual-changelog:
+  filename: CHANGELOG.md
+  markup-type: markdown
+license-output: LICENSE.md
+
+enable-nolib-creation: no
+embedded-libraries:
+  - Ace3
+  - callbackhandler
+  - libbase64-1-0
+  - libdatabroker-1-1
+  - libdbicon-1-0
+  - libstub
+  ## not published separately
+  # - tdGUI
+  # - LibCRC32-1.0
+  ### via tdGUI
+  - libwindow-1-1
```

Comment:
This list is hopefully adding attribution to those libraries on CurseForge. I was unable to test that, since I obviously couldn't push there (and didn't want to create a test project etc). If it doesn't work, that should only impact 
![image](https://user-images.githubusercontent.com/2866005/198557088-3382cc5f-d663-4d21-b2ba-63c3ea731c04.png)
this list, which is currently empty. It should properly link to all these addons afterwards, like in 
![image](https://user-images.githubusercontent.com/2866005/198557199-85a819ae-f8c6-4f9b-b9cd-a41b2571bf11.png)
or alike.